src/gui/windows/creative_inventory.zig

```diff
@@ -73,9 +73,22 @@ fn initContent() void {
 		const list = VerticalList.init(.{0, padding}, 144, 0);
 		items = .init(main.globalAllocator);
 		var itemIterator = main.items.iterator();
-		while(itemIterator.next()) |item| {
-			if(searchString.len != 0 and !std.mem.containsAtLeast(u8, item.id(), 1, searchString)) continue;
-			items.append(Item{.baseItem = item.*});
+		if(searchString.len > 1 and searchString[0] == '.') {
+			const bestTags = main.Tag.findSimilar(searchString[1..]);
+			defer bestTags.deinit();
+			while(itemIterator.next()) |item| {
+				for(bestTags.items) |tag| {
+					if(item.hasTag(tag) or (item.block() != null and (main.blocks.Block{.typ = item.block().?, .data = 0}).hasTag(tag))) {
```

Comment:
Better be explicit that you don't care about this field.
```suggestion
					if(item.hasTag(tag) or (item.block() != null and (main.blocks.Block{.typ = item.block().?, .data = undefined}).hasTag(tag))) {
```